components/mark.tsx

```diff
@@ -0,0 +1,10 @@
+export default function Mark(props) {
+  return (
+    <>
+      {" "}
+      <mark className="text-indigo-800 bg-indigo-100 rounded-md ring-indigo-100 ring-4 dark:ring-indigo-900 dark:bg-indigo-900 dark:text-indigo-200">
+        {props.children}
+      </mark>{" "}
+    </>
+  );
+}
```

Comment:
```suggestion
import {FC} from "react";

const Mark: FC = ({children}) => {
  return (
    <>
      {" "}
      <mark className="text-indigo-800 bg-indigo-100 rounded-md ring-indigo-100 ring-4 dark:ring-indigo-900 dark:bg-indigo-900 dark:text-indigo-200">
        {children}
      </mark>{" "}
    </>
  );
}

export default Mark
```

Reply:
Jeg har sat typescript type på

Reply:
Det Kan nok ogås godt slette de der tomme strings {""}

Reply:
Jeg tror de der {" "} er til for hvis man glemmer space rundt om det man marker